source/Templates/TeamCitySetup/.teamcity/octopus/main/MonoTestingProject.kt

```diff
@@ -0,0 +1,62 @@
+package octopus.main
+
+import jetbrains.buildServer.configs.kotlin.v2019_2.Project
+import jetbrains.buildServer.configs.kotlin.v2019_2.toId
+import octopus.main.buildtypes.MonoTestBuildType
+
+class MonoTestingProject : Project({
+    val projectName = "Mono Testing"
+    name = projectName
+
+    val buildTypesToRegister = sequence {
+        val items = listOf(
```

Comment:
I would seriously consider changing this to just say (using debian as an example) "just gimme a debian box, i dont care which version it is". That will be less maintenance down the road. That'll work well for all except ubuntu where we have multiple versions 🤔 

I've asked a [question in slack](https://octopusdeploy.slack.com/archives/C9TUPEJ8K/p1598397255019400) about someone taking ownership of the test matrix.

Reply:
So @matt-richardson would that be based on `system.Octopus.OS`?

Reply:
Yup

Reply:
@matt-richardson see https://github.com/OctopusDeploy/Sashimi/pull/98/commits/bca1ed052bb7d17a4a07e554097ed9d34585d33a, I think this is it right?

Reply:
Yeah, it'll work, but you'll hit some inconsistent builds with ubuntu... Sometimes it'll run on ubuntu 14, sometimes on 16, sometimes 18...

Reply:
Yeah, it'll work, but you'll hit some inconsistent builds with ubuntu... Sometimes it'll run on ubuntu 14, sometimes on 16, sometimes 18...

Reply:
This level of randomness may actually be a nice thing @matt-richardson 😄